### ISSUE TITLE
Compiled module lacks trailing semicolon

### ISSUE THREAD

**Issue by RobbinHabermehl** [goodeggs/angular-cached-resource#59]
The last line of the not minified compiled module lacks a trailing semicolon, which causes errors when concatenation is used:
```
},{"./modify_object_in_place":5,"./resource_cache_entry":10}]},{},[4])
```

A simple workaround is to specify this module as last file.

**Comment by demands** [comment]
Turns out the new version of browserify does this automatically :)
Thanks for the report!

**Comment by demands** [comment]
Fixed in [1.0.8](https://github.com/goodeggs/angular-cached-resource/releases/tag/v1.0.8).
